<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1402</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 154, 191);
background-color: rgb(253, 245, 223);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,1">
       <property name="spacing">
        <number>60</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>200</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnListDocuments">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>List Documents</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddDocuments">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Add Documents</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QCommandLinkButton" name="btnGoback">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 170, 127);</string>
         </property>
         <property name="text">
          <string>Go Back</string>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="widgetStack">
         <property name="minimumSize">
          <size>
           <width>1148</width>
           <height>670</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1200</width>
           <height>750</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(194, 154, 191);
background-color: rgb(253, 245, 223);</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="documents">
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>19</x>
             <y>70</y>
             <width>1111</width>
             <height>611</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_12">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Your Documents</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QTableWidget" name="tableDocument">
              <property name="minimumSize">
               <size>
                <width>900</width>
                <height>400</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>900</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 127);</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="addDocumentPage">
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>90</y>
             <width>1061</width>
             <height>351</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>12</number>
            </property>
            <property name="topMargin">
             <number>12</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>12</number>
            </property>
            <item alignment="Qt::AlignTop">
             <widget class="QLabel" name="label_6">
              <property name="styleSheet">
               <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="text">
               <string>Add a document</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="styleSheet">
               <string notr="true">border-radius: 4;
background-color: rgb(85, 170, 127);</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>8</number>
                 </property>
                 <property name="leftMargin">
                  <number>8</number>
                 </property>
                 <property name="topMargin">
                  <number>8</number>
                 </property>
                 <property name="rightMargin">
                  <number>8</number>
                 </property>
                 <property name="bottomMargin">
                  <number>8</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <property name="leftMargin">
                    <number>8</number>
                   </property>
                   <property name="topMargin">
                    <number>8</number>
                   </property>
                   <property name="rightMargin">
                    <number>8</number>
                   </property>
                   <property name="bottomMargin">
                    <number>8</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Title:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="inputTitle">
                     <property name="minimumSize">
                      <size>
                       <width>220</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>220</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="maxLength">
                      <number>25</number>
                     </property>
                     <property name="placeholderText">
                      <string>DocumentName</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <property name="leftMargin">
                    <number>8</number>
                   </property>
                   <property name="topMargin">
                    <number>8</number>
                   </property>
                   <property name="rightMargin">
                    <number>8</number>
                   </property>
                   <property name="bottomMargin">
                    <number>8</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Link:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="inputLink">
                     <property name="minimumSize">
                      <size>
                       <width>580</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="maxLength">
                      <number>255</number>
                     </property>
                     <property name="placeholderText">
                      <string>documentlink.com</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item alignment="Qt::AlignLeft">
                  <widget class="QPushButton" name="btnAdd">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
                   </property>
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
